Mathlib/Algebra/Order/Group/Abs.lean

```diff
@@ -145,21 +145,25 @@ theorem mabs_le_max_mabs_mabs (hab : a ≤ b) (hbc : b ≤ c) : |b|ₘ ≤ max |
     ⟨by simp [hbc.trans (le_mabs_self c)], by
       simp [(inv_le_inv_iff.mpr hab).trans (inv_le_mabs a)]⟩
 
+omit [IsOrderedMonoid G] in
```

Comment:
These indicate that they are in the wrong module. Something to address on clean up.

Reply:
At least one could add a `TODO` maybe?

Reply:
I think that grepping for `omit` is enough since any `omit` has the same signal.